### PR TITLE
jadx: Add version 1.1.0

### PR DESCRIPTION
**Pull request by Vixb1122**

### CODE REVIEW

File: bucket/jadx.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.1.0",
+    "description": "Dex to Java decompiler",
+    "homepage": "https://github.com/skylot/jadx",
+    "license": "Apache-2.0",
+    "url": "https://github.com/skylot/jadx/releases/download/v1.1.0/jadx-1.1.0.zip",
+    "hash": "91948067a60feee36512e043b288f726678ddd760aa11a01496b53df95465b16",
+    "bin": [
+        "bin\\jadx.bat",
+        "bin\\jadx-gui.bat"
```
Comment (Ash258) [inline bucket/jadx.json:10]: What are available parameters?

Comment (Vixb1122) [reply bucket/jadx.json:10]: You can see the available parameters via "https://github.com/skylot/jadx#usage", but usually the default parameters are sufficient.

Comment (Vixb1122) [reply bucket/jadx.json:10]: > What are available parameters?

You can see the available parameters via "https://github.com/skylot/jadx#usage", but usually the default parameters are sufficient.

**Pull request merged**